src/utils/store.js

```diff
@@ -1,3 +1,22 @@
+function canUseStorage(type) {
+  let storage;
+  try {
+    storage = window[type];
+    const x = '__storage_test__';
+    storage.setItem(x, x);
+    storage.removeItem(x);
+    return true;
+  } catch (e) {
+    return (
+      e instanceof DOMException
+      && e.name === 'QuotaExceededError'
```

Comment:
Why do we make an exception for this error only?

Reply:
This was taken from here.

https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#testing_for_availability

I am guessing that if the Quota is Exceeded, storage is still technically available. Just full.